src/config/networkConfig.ts

```diff
@@ -272,13 +278,105 @@ const networks: { [K in NetworkIds]: Network } = {
     ],
     realityTimeout: 10,
   },
+  [NetworkIds.XDAI]: {
+    earliestBlockToCheck: EARLIEST_XDAI_BLOCK_TO_CHECK,
+    contracts: {
+      conditionalTokensAddress: CONDITIONAL_TOKEN_CONTRACT_ADDRESS_FOR_XDAI,
+      realityAddress: REALITY_CONTRACT_ADDRESS_FOR_XDAI,
+      wrapped1155FactoryAddress: WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_XDAI,
+      cpk: {
+        masterCopyAddress: '0x6851D6fDFAfD08c0295C392436245E5bc78B0185',
+        proxyFactoryAddress: '0xfC7577774887aAE7bAcdf0Fc8ce041DA0b3200f7',
+        multiSendAddress: '0x035000FC773f4a0e39FcdeD08A46aBBDBF196fd3',
+        fallbackHandlerAddress: '0x602DF5F404f86469459D5e604CDa43A2cdFb7580',
+      },
```

Comment:
This should make the cpk works in xDai. But more research is needed.

Reply:
Let's take a look https://github.com/gnosis/contract-proxy-kit/blob/development/src/config/networks.ts#L73